### ISSUE TITLE
Update to use eos-downloader

### ISSUE THREAD

**Issue by ryanmerolle** [ipspace/netlab#580]
Depending on what you think on the approach we have two options:

1.) Update documentation to mention [eos-download](https://github.com/titom73/eos-downloader) (installable via pip) can be downloaded with the tool.

```bash
eos-download --image cEOS --version 4.28.3M --import_docker --token fdfdgdghfhhfh
```

2.) Add eos-download to the workflow for vm and docker images for Arista.

```bash
eos-download --image cEOS --version 4.28.3M --import_docker --token fdfdgdghfhhfh
```

I'm happy to handle the documentation even if you select option 2.

**Comment by ryanmerolle** [comment]
I should have said this, but this also works for veos

**Comment by ipspace** [comment]
Thank you, interesting tool!

I would go for option 1 -- I got burned too many times when I described how to use a tool and then either the tool changed (hello, Ansible) or it didn't work and someone expected me to help him fix that.

Would you add a mention to the documents describing how to download a cEOS container or build a vEOS box and submit a PR?

**Comment by ryanmerolle** [comment]
I will take care of this in a PR.  Thanks for the follow up.

**Comment by ipspace** [comment]
Implemented in #602
